src/provider/model/Model.ts

```diff
@@ -7,9 +7,10 @@ import {Inject} from '../common/Injectables';
 import {ChannelId, DEFAULT_CHANNEL_ID} from '../../client/main';
 import {APIHandler} from '../APIHandler';
 import {APIFromClientTopic} from '../../client/internal';
-import {SYSTEM_CHANNELS, Timeouts} from '../constants';
+import {SYSTEM_CHANNELS, Timeouts, CustomConfigFields} from '../constants';
 import {withStrictTimeout, untilTrue, allowReject, untilSignal} from '../utils/async';
 import {Boxed} from '../utils/types';
+import {checkCustomConfigField} from '../utils/helpers';
```

Comment:
Sorry, one last thing! Just some unused imports in this file

Reply:
good catch thanks